static/app/components/searchQueryBuilder/tokens/filterKeyListBox/index.tsx

```diff
@@ -333,19 +359,23 @@ export function FilterKeyListBox<T extends SelectOptionOrSectionWithKey<string>>
   const areAiFeaturesAllowed =
     !organization?.hideAiFeatures && organization.features.includes('gen-ai-features');
 
-  // Add recent filters to hiddenOptions so they don't show up the ListBox component.
-  // We render recent filters manually in the RecentFiltersPane component.
-  const hiddenOptionsWithRecentsAdded = useMemo<Set<SelectKey>>(() => {
-    return new Set([
+  const hiddenOptionsWithRecentsAndAskSeerAdded = useMemo<Set<SelectKey>>(() => {
+    const baseHidden = [
       ...hiddenOptions,
       ...recentFilters.map(filter => createRecentFilterOptionKey(getKeyName(filter.key))),
-    ]);
-  }, [hiddenOptions, recentFilters]);
+    ];
+
+    if (traceExploreAiQueryContext && areAiFeaturesAllowed) {
+      baseHidden.push('ask_seer');
```

Comment:
We refer to this `ask_seer` value a few times in different places, can we make it a constant `ASK_SEER_ITEM_KEY`?